pilot/pkg/model/validation.go

```diff
@@ -1775,8 +1790,8 @@ func validateGatewayNames(gateways []string) (errs error) {
 		parts := strings.SplitN(gateway, "/", 2)
 		if len(parts) != 2 {
 			// deprecated
-			log.Warn("Gateway names with FQDN format or short forms are deprecated. " +
-				"Use namespace/name format instead")
+			// log.Warn("Gateway names with FQDN format or short forms are deprecated. " +
+			//	"Use namespace/name format instead")
```

Comment:
do you mean to leave this commented out warning?

Reply:
It's extremely verbose - and having it logged 10000 times without anyone looking will just drawn legitimate warnings.

Plus you have no clue which gateway you need to edit - just that one of the namespaces has a bad gateway, somewhere.